server/contracts/NFTStorefront.cdc

```diff
@@ -0,0 +1,495 @@
+import FungibleToken from "./FungibleToken.cdc"
```

Comment:
It would be cool if we could also import this one through a go module like we do the `NonFungibleToken` one 🤔.

That would also mean that the emulator doesn't need the import address replacement logic.

Reply:
Yeah, it would!! But maybe later? It'd just require making a few changes to the `nft-marketplace` repo, to make it friendlier to use as a module. It'd be really neat to make some kind of interface for structuring new contract repos so that they can be imported easily!! (Maybe like how fvm imports the FungibleToken and other contracts!)

On the plus side, by including this file it shows how easy it is to add your own contracts ;)